deployers/src/main/java/org/ironjacamar/deployers/common/AbstractResourceAdapterDeployer.java

```diff
@@ -274,12 +276,28 @@ public Deployment activate(Connector connector, Activation activation, ClassLoad
          builder.classLoaderPlugin(classLoaderPlugin);
 
          if (connector.getResourceadapter().getResourceadapterClass() != null)
+         {
```

Comment:
The current BootstrapContext integrations needs to be taken out - use a method variable instead of setBootstrapContext.

You could inject the coordinator instance - eliminates the need for a singleton.